pallets/drand/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "pallet-drand"
 description = "FRAME pallet for briding to drand."
-authors = ["Tony Riemer <driemworks@idealabs.network>"]
+authors = [
+	"Tony Riemer <driemworks@idealabs.network>",
+	"Juan Girini <juan@idealabs.network>"
+	]
 version = "0.0.1"
 license = "MIT-0"
```

Comment:
Unrelated, but I created https://github.com/ideal-lab5/ideal-network/issues/13 to track work related to fixing license inconsistency issues. I'm not sure why I put MIT-0 here, I think we should aim for Apache-2.0 everything.